### PR TITLE
python-websocket-client: include homepage

### PR DESCRIPTION
**Pull request by A108384**
**Summary**
- Add homepage to package.yml

**Test Plan**
- Checked that homepage was added.  Created python test program.  Was able to import and run a test call via python.

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/py/python-websocket-client/pspec_x86_64.xml [hunk 1]
```diff
@@ -57,12 +58,12 @@
...
-        <Update release="14">
-            <Date>2024-02-15</Date>
+        <Update release="15">
+            <Date>2024-06-30</Date>
             <Version>1.6.1</Version>
             <Comment>Packaging update</Comment>
-            <Name>Joey Riches</Name>
-            <Email>josephriches@gmail.com</Email>
+            <Name>Clifford Red</Name>
+            <Email>A108384@GMAIL.COM</Email>
```
Comment (malfisya) [inline packages/py/python-websocket-client/pspec_x86_64.xml:66]: Why this is all capital letter?

**Review by malfisya — CHANGES_REQUESTED** [review]
Hey, thank you for the pull request. A few things:
- Please squash your commit. There is an extra commit in your PR. 
- Is your mail address valid? It usually doesn't use all capital letter.

**Comment by A108384** [comment]
Yes, my email is valid. It is just the way I format it... I can change to
lower case for this.
I'll also try to squash my commits.

Thanks for the feedback.
-cliff

File: packages/py/python-websocket-client/pspec_x86_64.xml (hunk 1 above)
Comment (A108384) [reply packages/py/python-websocket-client/pspec_x86_64.xml:66]: Do I actually need to change the capitalization of my email address or is this just a question?
If I need to change it, where does this come from?  When I rebuild the package it is updated automatically.

Also - can you tell me the best way to consolidate these commits?

Sorry - I am still trying to figure out git and github.   I would appreciate your patience.

**Pull request closed without merging**